run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+failed=0
+
+build_all() {
+  sh -c 'rush build \
+    -t truffle-tutorial-hello-world \
+    -t truffle-tutorial-echo \
+    -t truffle-tutorial-token \
+    -t truffle-tutorial-nft'
+}
+
+rebuild_all() {
+  sh -c 'rush rebuild \
+    -t truffle-tutorial-hello-world \
+    -t truffle-tutorial-echo \
+    -t truffle-tutorial-token \
+    -t truffle-tutorial-nft'
+}
+
+test_all() {
+  examples=(
+    "hello-world"
+    "echo"
+    "token"
+    "nft"
```

Comment:
Should these be prefixed with `truffle-tutorial-` or will they execute as expected no matter what?

Reply:
they are folder names, so we will loop these and cd them (at line 35)

Reply:
so they need to match the actual folder name